src/features/usage-consolidation-accordion/usage-consolidation-content-resource/usage-consolidation-content-resource.test.js

```diff
@@ -0,0 +1,116 @@
+import { render } from '@testing-library/react';
+
+import Harness from '../../../../test/jest/helpers/harness';
+
+import UsageConsolidationContentResource from './usage-consolidation-content-resource';
+import { costPerUseTypes } from '../../../constants';
+
+const costPerUseData = {
+  data: {
+    [costPerUseTypes.RESOURCE_COST_PER_USE]:{
+      attributes: {
+        analysis: {
+            cost: 1,
+            costPerUse: 1,
+            usage: 1
+        },
+        parameters: {
+            currency: '',
+            startMonth: '',
+        },
+        usage:{
+            platforms:[
+                {
+                    counts: [3, 6, 16],
+                    isPublisherPlatform: true,
+                    name: "name",
+                    total: 17
+                }
```

Comment:
When we have multiline objects or arrays, we should put trailing commas after the last item, so please add them here and in all other places of this file :)

```suggestion
                    total: 17,
                },
```

Reply:
Implemented the changes in latest commit